module-1/lab-api-scavenger-game/your-code/API_Challenge_2.ipynb

```diff
@@ -0,0 +1,1921 @@
+{
```

Comment:
In this challenge you can see the importance of normalizing JSONs.
You have not included the count of commits between the 2 dates filtered.